### PR TITLE
certdog: remove dependency on model

### PR DESCRIPTION
**Pull request by jmt-lab**
**Description of changes:**

Removes model dependency from certdog and tokio which is unnecessary

**Testing done:**

Launched and validated template

**Terms of contribution:**

By submitting this pull request, I agree that this contribution is dual-licensed under the terms of both the Apache License, version 2.0, and the MIT license.

### CODE REVIEW

**Comment by jmt-lab** [comment]
Fixed forgotten removal of model:: from test module

**Review by webern — APPROVED** [review]
:tada:

File: sources/api/certdog/Cargo.toml [hunk 1]
```diff
@@ -10,16 +10,14 @@ build = "build.rs"
...
 
 [dependencies]
-apiclient = { path = "../apiclient", version = "0.1" }
 argh = "0.1"
 base64 = "0.13"
 constants = { path = "../../constants", version = "0.1" }
 log = "0.4"
-models = { path = "../../models", version = "0.1" }
-serde_json = "1"
+modeled-types = { path = "../../models/modeled-types" }
```
Comment (webern) [inline sources/api/certdog/Cargo.toml]:
```suggestion
modeled-types = { path = "../../models/modeled-types", version = "0.1"  }
```

```
6 │ modeled-types = '*'
  │ ^^^^^^^^^^^^^^^^^^^ wildcard crate entry
```

File: sources/api/certdog/Cargo.toml [hunk 2]
```diff
@@ -10,16 +10,14 @@ build = "build.rs"
...
 
 [dependencies]
-apiclient = { path = "../apiclient", version = "0.1" }
 argh = "0.1"
 base64 = "0.13"
 constants = { path = "../../constants", version = "0.1" }
 log = "0.4"
-models = { path = "../../models", version = "0.1" }
-serde_json = "1"
+modeled-types = { path = "../../models/modeled-types", version = "0.1"  }
```
Comment (ecpullen) [inline sources/api/certdog/Cargo.toml]: nit: extra space

**Review by ecpullen — APPROVED** [review]

**Review by webern — APPROVED** [review]

**Comment by jmt-lab** [comment]
Squashed commits

**Pull request merged**